capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameavle.correct_name.capitalize
+  end
```

Comment:
- [ ] Kindly fix the typographical error you have here. It should be `@nameable` and not `@nameavle`. 👍